daprdocs/content/en/reference/components-reference/supported-state-stores/setup-coherence.md

```diff
@@ -0,0 +1,136 @@
+---
+type: docs
+title: "Coherence"
+linkTitle: "Coherence"
+description: Detailed information on the Coherence state store component
+aliases:
+  - "/operations/components/setup-state-store/supported-state-stores/setup-coherence/"
+---
+
+## Component format
+
+To setup Coherence state store create a component of type `state.coherence`. See [this guide]({{< ref "howto-get-save-state.md#step-1-setup-a-state-store" >}}) on how to create and apply a state store configuration.
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: <NAME>
+spec:
+  type: state.coherence
```

Comment:
This is an Oracle DB yes? Should this type then be `state.oracle.coherence` or `state.oci.coherence`

Reply:
This is not part of Oracle DB, and not dependant on it, but it is an Oracle product.
In components-contrib i have already committed as:
```
  type: state.coherence
```

thoughts?